WordPress/Jetpack/Classes/Utility/DataMigrator.swift

```diff
@@ -115,15 +122,30 @@ private extension DataMigrator {
         return true
     }
 
-    func copyUserDefaults(from source: UserDefaults?, to destination: UserDefaults?) -> Bool {
-        guard let source, let destination else {
+    func populateSharedDefaults() -> Bool {
+        guard let sharedDefaults = sharedDefaults else {
```

Comment:
```suggestion
        guard let sharedDefaults else {
```
We can write this cool line now 😎

Reply:
Oh right, I keep forgetting it